### PR TITLE
Publish `utxo-db.pdf` and `utxo-db-api.pdf` docs on the website

### PR DESCRIPTION
**Pull request by jorisdral**

### CODE REVIEW

File: docs/pdfs/report/.gitignore
Comment (dnadales) [inline docs/pdfs/report/.gitignore:1]: What about using `reports` or `tech-reports` for our technical reports. The name `pdfs` might not be very representative of that folder's contents, and would become a misnomer if we write a report in other formats (although it's unlikely I think).

File: docs/website/sidebars.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const sidebars = {
       label: 'About Ouroboros',
       items: [
         'about-ouroboros/index',
+        'about-ouroboros/utxo-hd',
```
Comment (dnadales) [inline docs/website/sidebars.js:26]: Aren't the reports too low-level for this section? I would expect these to be linked from the "For Developers" section.

File: docs/pdfs/report/.gitignore
Comment (jorisdral) [reply docs/pdfs/report/.gitignore:1]: Done! Moved the files

File: docs/website/sidebars.js (hunk 1 above)
Comment (jorisdral) [reply docs/website/sidebars.js:26]: Also, these are only references. The contents of the `.md` file shouldn't be too technical, but I think it's fine to link documents for further reading, so I don't think they have to move

Comment (dnadales) [reply docs/website/sidebars.js:26]: > The main report is also listed in `about-ouroboros/References`. Should we move that one too then?

I think that would make sense.

Comment (dnadales) [reply docs/website/sidebars.js:26]: In other words my main objection is that it feels strange to have a top level (sidebar) link for a very specific feature.

**Review by dnadales — APPROVED** [review]

**Pull request merged**